server/src/main/java/io/crate/analyze/expressions/ExpressionAnalysisContext.java

```diff
@@ -41,6 +41,7 @@ public class ExpressionAnalysisContext {
 
     private boolean hasAggregates;
     private boolean allowEagerNormalize = true;
+    private boolean isInPredicate = false;
```

Comment:
I'd tend to rename this to something like `parentIsOrderSensitive` (or insensitive) as it gives it a different semantic meaning that is more general applicable. It also communicates a bit more why this information may be useful

Reply:
yep, makes sense, thx!